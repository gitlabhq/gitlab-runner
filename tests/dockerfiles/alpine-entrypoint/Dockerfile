FROM alpine:latest

RUN apk --no-cache add su-exec

COPY alpine-entrypoint/entrypoint.sh /entrypoint

ENTRYPOINT ["/entrypoint"]
