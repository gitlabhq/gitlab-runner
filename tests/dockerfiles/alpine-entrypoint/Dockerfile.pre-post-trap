FROM alpine:latest

RUN apk --no-cache add bash

COPY alpine-entrypoint/entrypoint.pre-post-trap.sh /entrypoint

# non-root user required to enable the docker executor's file ownership change
USER 1000:1000

ENTRYPOINT [ "/entrypoint" ]
