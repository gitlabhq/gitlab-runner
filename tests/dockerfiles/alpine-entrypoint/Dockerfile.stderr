FROM alpine:latest

COPY alpine-entrypoint/entrypoint-stderr.sh /entrypoint

# non-root user required to enable the docker executor's file ownership change
USER 1000:1000

ENTRYPOINT ["/entrypoint"]
